cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -47,21 +47,21 @@ These functions return a single value.
 [options="header"]
 |===
 | Function                                                            | Description
-| <<functions-coalesce,coalesce()>>                                   | Returns the first non-`null` value in the list of expressions passed to it.
-| <<functions-endnode,endNode()>>                                     | Returns the last node of a relationship.
+| <<functions-coalesce,coalesce()>>                                   | Returns the first non-`null` value in a given list of expressions.
+| <<functions-endnode,endNode()>>                                     | Returns the end node of a relationship.
 | <<functions-head,head()>>                                           | Returns the first element in a list.
-| <<functions-id,id()>>                                               | Returns the id of the relationship or node.
+| <<functions-id,id()>>                                               | Returns the id of a relationship or node.
 | <<functions-last,last()>>                                           | Returns the last element in a list.
 | <<functions-length,length()>>                                       | Returns the length of a path.
-| <<functions-properties,properties()>>                               | If the argument is a node or a relationship, the returned map is a map of its properties.
+| <<functions-properties,properties()>>                               | Returns a map containing all the properties of a node or relationship.
 | <<functions-size,size()>>                                           | Returns the number of items in a list.
 | <<functions-size-of-pattern-expression,Size of pattern expression>> | Returns the number of sub-graphs matching the pattern expression.
 | <<functions-size-of-string,Size of string>>                         | Returns the size of a string.
-| <<functions-startnode,startNode()>>                                 | Returns the first node of a relationship.
+| <<functions-startnode,startNode()>>                                 | Returns the start node of a relationship.
 | <<functions-timestamp,timestamp()>>                                 | Returns the difference, measured in milliseconds, between the current time and midnight, January 1, 1970 UTC.
-| <<functions-toboolean,toBoolean()>>                                 | Converts the argument to a boolean and returns the result.
-| <<functions-tofloat,toFloat()>>                                     | Converts the argument to a float and returns the result.
-| <<functions-tointeger,toInteger()>>                                 | Converts the argument to an integer and returns the result.
+| <<functions-toboolean,toBoolean()>>                                 | Casts a string value to a boolean value.
```

Comment:
See my comments above, I think 'cast' implies a very non-invasive change, but 'converts' or at least 'coerces' is more appropriate in these cases.